source/configuration/modules/omstdout.rst

```diff
@@ -16,21 +19,96 @@ the quality of this module as we do not expect it to be used in real
 deployments. If you do, please drop us a note so that we can enhance
 its priority!
 
-**Configuration parameters**:
 
-Note: configuration directive names are case-insensitive.
+Configuration
+=============
+
+.. note::
+
+   Parameter names are case-insensitive.
+
+
+Module Parameter
+----------------
+
+none
+
+
+Action Parameter
+----------------
+
+template
+^^^^^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "word", "RSYSLOG_FileFormat", "no", "none"
+
+Set the template which will be used for the output. If none is specified
+the default will be used.
+
+
+EnsureLFEnding
+^^^^^^^^^^^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "binary", "on", "no", "``$ActionOMStdoutEnsureLFEnding``"
+
+Makes sure, that each essage is written with a terminating LF. If the
+message contains a trailing LF, none is added. This is needed for the
+automated tests.
 
--  **$ActionOMStdoutArrayInterface** [on\|**off**
+
+Configure statement
+-------------------
+
+This is used when building rsyslog from source.
+
+./configure --enable-omstdout
+
+
+Legacy parameter not adopted in the new style
+---------------------------------------------
+
+-  **$ActionOMStdoutArrayInterface**
+   [Default: off]
    This setting instructs omstdout to use the alternate array based
    method of parameter passing. If used, the values will be output with
    commas between the values but no other padding bytes. This is a test
    aid for the alternate calling interface.
--  **$ActionOMStdoutEnsureLFEnding** [**on**\ \|off
-   Makes sure that each message is written with a terminating LF. This
-   is needed for the automatted tests. If the message contains a
-   trailing LF, none is added.
 
-**Caveats/Known Bugs:**
 
-Currently none known.
+Examples
+========
+
+Minimum setup
+-------------
+
+The following sample is the minimum setup required to have syslog messages
+written to stdout.
+
+.. code-block:: none
+
+   module(load="omstdout")
+   action(type="omstdout")
+
+
+Example 2
+---------
+
+The following sample will write syslog messages to stdout, using a template
+and input from imtcp.
+
```

Comment:
I'm probably overlooking it, but how does imtcp factor into the example?

Reply:
Sry, I took imtcp out of the example to keep it as short as possible but forgot to change the description